src/components/business/WhatCanWeDo.tsx

```diff
@@ -78,29 +89,48 @@ export const WhatCanWeDo = ({ textContent }: WhatCanWeDoProps): JSX.Element => {
         </div>
 
         {/*Mobile/Tablet View*/}
-        <div className="flex w-full snap-x snap-mandatory flex-row justify-start gap-6 overflow-scroll  xl:hidden">
-          {textContent.cards.map((testimonial, index) => (
-            <div
-              key={testimonial.selectorTab}
-              className="flex w-full shrink-0 snap-center flex-col justify-end rounded-3xl p-2"
-            >
-              <button
-                className={`flex ${
-                  selectedTab === index ? 'border-primary' : 'border-transparent'
-                } w-full max-w-[300px] translate-y-1 flex-col items-center justify-center border-b-4 p-5 text-center`}
-                onClick={() => {
-                  onTabSelectorButtonClicked(index);
-                }}
+        <div className="relative w-full snap-x snap-mandatory flex-row justify-start gap-6 lg:hidden">
+          <div className="flex w-full snap-x snap-mandatory space-y-5">
+            {textContent.cards.map((testimonial) => (
+              <div
+                key={testimonial.selectorTab}
+                className="rounded- flex w-full shrink-0 snap-center flex-col justify-end px-5"
               >
-                <p className="text-center text-3xl font-medium text-white">{testimonial.selectorTab}</p>
-              </button>
-              <div className="items- flex h-full flex-col space-y-10" key={testimonial.review}>
-                <div className="flex w-full max-w-[890px] flex-col">
-                  <p className="text-center text-xl text-white">{testimonial.description}</p>
+                <div className="flex h-full flex-col">
+                  <div className="flex min-h-[110px] w-full max-w-[890px] flex-col items-end justify-end">
+                    <p className="py-5 text-center text-3xl font-medium text-white">
+                      {textContent.cards[selectedTab].selectorTab}
+                    </p>
+                  </div>
+
+                  <div className="flex flex-row items-center justify-center space-x-2 py-4">
+                    {selectedTab !== 0 && (
+                      <button onClick={onLeftArrowClick} className="flex items-center text-white">
+                        <CaretLeft size={24} />
+                        <span className="ml-2">{textContent.betweenDots}</span>
+                      </button>
+                    )}
+
+                    <button
+                      onClick={onRightArrowClick}
+                      disabled={selectedTab === textContent.cards.length - 1}
```

Comment:
Can you extract this conditional to a constant?

Reply:
Done